### DISCUSSION TITLE
👨‍🎨 New Logo

### DISCUSSION THREAD

**Discussion by N3v1** [TeamCodeArena/DeliverEase#114] [Polls]
Here you can vote for the new DeliverEase logo

> **NOTE:**\
> Please hold off on voting until we're finished. I'll give the signal to start.

## Which options can you vote?
| A | B | C |  D | E |
|--------|--------|--------|-------|--------|
| <img width="30" src="https://github.com/TeamCodeArena/DeliverEase/assets/129311622/831ef6d2-7666-4de1-9d9c-63a8d05f06d8"> |  <img width="30" src="https://github.com/TeamCodeArena/DeliverEase/assets/129311622/e273ae19-8f26-4f7e-a46e-137a2199679f"> | <img width="30" src="https://github.com/TeamCodeArena/DeliverEase/assets/129311622/93ff13c2-8dad-49f1-aed1-f5bb50f1768e"> | <img width="30" src="https://github.com/TeamCodeArena/DeliverEase/assets/129311622/1a7ec4d8-c859-498e-8b03-d8e3f5f57f22">| <img width="30" src="https://github.com/TeamCodeArena/DeliverEase/assets/129311622/d8577b4a-baea-4f43-a1d6-5cfbf5165d33">|

>**Note:**\
> There will be several rounds

cc: @Devansh-Singh-Parmar @SID262000 @Atrika10 @AryanGupta0007

**Comment by Atrika10** [comment]
option A & B is not getting matched with our frontend color combination so I'll go with 3rd option.

**Reply by N3v1** [reply]
Okay @Atrika10 We might have to change the black background to a darker gray because the background of our product page is black - the Logo is not visible

**Reply by Atrika10** [reply]
yep, will Recheck @N3v1 👍

**Comment by Devansh-Singh-Parmar** [comment]
Hey @N3v1 
can you give it a touch of our website theme?

**Reply by N3v1** [reply]
![SCR-20230904-phzf](https://github.com/TeamCodeArena/DeliverEase/assets/129311622/6fc0c61d-61f6-4c75-942e-f2543e8b36e4)

This is a part of the homepage @Devansh-Singh-Parmar 
See design here: https://www.figma.com/file/jSIuBlHIQVMXDyERjeWyi7/Untitled?type=design&node-id=0%3A1&mode=design&t=gsObqmcijy78ox7s-1

**Comment by AryanGupta0007** [comment]
No make the color scheme of the logo as per the design scheme of the website . @N3v1 @SID262000

**Reply by SID262000** [reply]
I agree with @AryanGupta0007 
cc: @N3v1

**Comment by N3v1** [comment]


**Reply by AryanGupta0007** [reply]
> The winner is Option A 🥳🎉
> ![IMG_8304](https://github.com/TeamCodeArena/DeliverEase/assets/129311622/b19e6f18-b4e7-4cb8-9390-f22d9fff0fd3)
> 

But don't add that. @N3v1

**Reply by SID262000** [reply]
Why @AryanGupta0007 
cc: @N3v1

**Reply by AryanGupta0007** [reply]
@SID262000 doesn't match with the design scheme of our website.

**Reply by SID262000** [reply]
Please work with @N3v1 to get this sorted at the top priority!

**Reply by N3v1** [reply]
@Devansh-Singh-Parmar and I created some more logos. When we are done I'll update this poll
cc: @SID262000 @Devansh-Singh-Parmar @Atrika10 @AryanGupta0007

**Comment by N3v1** [comment]
Hi everyone,
Round 2 of voting 🗳️ 
Please start when I give my 🟢 signal

cc: @SID262000 @Devansh-Singh-Parmar @Atrika10 @AryanGupta0007

**Reply by AryanGupta0007** [reply]
Voting is not needed you need to make the logo as per the design scheme of the website and also note that most of the voters might have never seen our project so voting has no meaning. 
Cc: @N3v1 @SID262000

**Reply by SID262000** [reply]
@N3v1 I request you to kindly co-ordinate with @AryanGupta0007 to get this issue resolved at top priority. If required, I would suggest to stop voting to avoid unnecessary delays in the progress, Thanks!

**Reply by N3v1** [reply]
Okay @AryanGupta0007 Which of these logos do you like? Which gradients do you like, give me some examples of what you like?

cc: @SID262000 @Devansh-Singh-Parmar

**Reply by N3v1** [reply]
Choose an option that you like the icon best @AryanGupta0007 

cc: @SID262000 @Devansh-Singh-Parmar

**Reply by AryanGupta0007** [reply]
@N3v1 will connect with you tommorow on slack

**Reply by N3v1** [reply]
Please note that I will be very busy in the next few weeks and have a lot to do. So please DM me. Please also contact @Devansh-Singh-Parmar, he helped me create some of the logos mentioned above. @AryanGupta0007 
cc: @SID262000 @Devansh-Singh-Parmar

**Reply by N3v1** [reply]
Please also note the time difference - IST -> CET/CET -3.30h @AryanGupta0007 

Example: 3.30pm IST --3.3h —> 12:00 CET/MEZ noon

**Reply by SID262000** [reply]
Thanks for the update, @N3v1 . Many Thanks for all your contributions! Keep us posted once you're back from your busy schedule.
In the meantime, folks please make sure to contact @N3v1 for anything required from his end by Sunday, post that he won't be available as indicated in the thread, Thanks!

**Reply by N3v1** [reply]
You can reach me at any time, but I will probably reply late - I will check Slack every other day. From next week I will not be available from Monday to Friday. 

cc: @SID262000 @Devansh-Singh-Parmar @AryanGupta0007

**Reply by SID262000** [reply]
Sure thing, @N3v1
